04_setup_ironic.sh

```diff
@@ -49,6 +49,14 @@ for IMAGE_VAR in $(env | grep "_LOCAL_IMAGE=" | grep -o "^[^=]*") ; do
     echo "RUN sed -i 's%$OLDIMAGE%${!IMAGE_VAR}%g' /release-manifests/*" >> $DOCKERFILE
 done
 
+if [ ! -z "${MIRROR_IMAGES}" ]; then
+    oc adm release mirror \
+       --insecure=true \
+        -a $REGISTRY_AUTH_FILE  \
+        --from $OPENSHIFT_RELEASE_IMAGE \
+        --to 192.168.111.1:5000/localimages/local-release-image
```

Comment:
Could we make it parametrized via `MIRROR_IMAGES` variable, which if set points to IP/NAME of registry?
```suggestion
        --to ${MIRROR_IMAGES}/localimages/local-release-image
```

Reply:
Yeah, the local registry IP address seems to be hardcoded all over the place.

Reply:
Done in #853 and this PR rebased on master